assets/src/edit-story/components/library/panes/text/textPresets.js

```diff
@@ -46,44 +47,47 @@ const PRESETS = [
   {
     title: __('Heading 1', 'web-stories'),
     element: {
-      content: `<span style="font-weight: 700">${__(
+      content: `<span style="font-weight: ${FONT_WEIGHT.BOLD}">${__(
         'Heading 1',
         'web-stories'
       )}</span>`,
-      fontSize: dataFontEm(2),
-      lineHeight: 1.5,
+      fontWeight: FONT_WEIGHT.BOLD,
+      fontSize: dataFontEm(2.7),
+      lineHeight: 1.1,
       x: DEFAULT_LEFT_MARGIN,
-      y: (PAGE_HEIGHT - dataFontEm(2)) / 2,
+      y: (PAGE_HEIGHT - dataFontEm(2.7)) / 2,
       font: TEXT_ELEMENT_DEFAULT_FONT,
       width: DEFAULT_ELEMENT_WIDTH,
     },
   },
   {
     title: __('Heading 2', 'web-stories'),
     element: {
-      content: `<span style="font-weight: 600">${__(
+      content: `<span style="font-weight: ${FONT_WEIGHT.BOLD}">${__(
         'Heading 2',
         'web-stories'
       )}</span>`,
-      fontSize: dataFontEm(1.5),
-      lineHeight: 1.5,
+      fontWeight: FONT_WEIGHT.BOLD,
+      fontSize: dataFontEm(2),
+      lineHeight: 1.2,
       x: DEFAULT_LEFT_MARGIN,
-      y: (PAGE_HEIGHT - dataFontEm(1.5)) / 2,
+      y: (PAGE_HEIGHT - dataFontEm(2)) / 2,
       font: TEXT_ELEMENT_DEFAULT_FONT,
       width: DEFAULT_ELEMENT_WIDTH,
     },
   },
   {
     title: __('Heading 3', 'web-stories'),
     element: {
-      content: `<span style="font-weight: 500">${__(
+      content: `<span style="font-weight: ${FONT_WEIGHT.BOLD}">${__(
```

Comment:
It looks like `500` (Medium) exists for Roboto? Or is the 500 => 700 change intentional?

Reply:
Yes, it's intentional as so far as the figma docs simply list `Bold` as the font-weight for all 3 headers:
![image](https://user-images.githubusercontent.com/40646372/91478019-0fe84a80-e854-11ea-8fa2-344130f38b95.png)